graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -12,9 +13,24 @@
                 # fraction of results that pass the filter
 ))
 
+# IntegerInterval is used to denote a continuous non-empty interval of integers.
+IntegerInterval = namedtuple('IntegerInterval', (
+    'lower_bound',    # optional int, inclusive lower bound of integers in the interval.
+                      # Intervals that do not have a lower bound denote this by using None.
+    'upper_bound',    # optional int, inclusive upper bound of integers in the interval.
+                      # Intervals that do not have an upper bound denote this by using None.
+))
```

Comment:
I like this definition. You could have used a tuple, but then you'd have to document what it means everywhere :)